apps/src/applab/applab.js

```diff
@@ -855,7 +853,7 @@ Applab.init = function (config) {
       localeDirection={studioApp.localeDirection()}
       editCode={!!config.level.editCode}
       readonlyWorkspace={Applab.reduxStore.getState().level.isReadOnlyWorkspace}
-      extraControlRows={extraControlRows}
+      showDebugger={showDebugButtons || showDebugConsole}
```

Comment:
Another option instead of passing this in would be to always render JsDebugger, and have JsDebugger render an empty div in the case it's not really wanted. Not sure which is better. Thoughts?